Makefile

```diff
@@ -12,6 +14,7 @@ dev:
 	dune build @src/fmt --auto-promote || true
 	dune build --profile dev
 	dune build @doc
+	dune build src/grvweb/{main.bc.js,index.html,style.css,viz.js,lite.render.js}
```

Comment:
instead of individually naming each file here, put them all in a `www` directory like we do in hazel and move that

Reply:
We only need here index.html and style.css here